spanner/api/Spanner.Samples.Tests/UpdateUsingDmlReturningAsyncTest.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Spanner.Data;
+using System.Threading.Tasks;
+using Xunit;
+
+[Collection(nameof(SpannerFixture))]
+public class UpdateUsingDmlReturningAsyncTest
+{
+    private readonly SpannerFixture _spannerFixture;
+
+    public UpdateUsingDmlReturningAsyncTest(SpannerFixture spannerFixture)
+    {
+        _spannerFixture = spannerFixture;
+    }
+
+    [Fact]
+    public async Task TestUpdateUsingDmlReturningAsync()
+    {
+        await _spannerFixture.RunWithTemporaryDatabaseAsync(async databaseId =>
```

Comment:
Why is this test different to all the others?

Reply:
This is because DML Returning Sample for Update is different from Insert and Delete. Ref comment https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/2088#issuecomment-1465090245

Reply:
Okay, we can leave it for now - but I think it's something else that should go on the backlog to fix later. The more consistent we can make this, the less time it'll take to maintain these samples.